website/content/docs/commands/workers/list.mdx

```diff
@@ -40,5 +40,11 @@ The default value is `false`.
 - `scope-id=<string>` - The scope from which to list the workers.
 The default value is `global`.
 You can also specify this value using the **BOUNDARY_SCOPE_ID** environment variable.
+- `list_token` (optional) - An opaque token that is returned from the previous `list` response.
+If you do not supply a value, pagination starts from the beginning.
+- `page_size` (optional) - An unsigned integer that indicates the number of items that should be included on the page of search results.
+If you do not provide a value, or if you provide a value of `0`, Boundary uses the default page size of 1000 items.
+If you configure it, the `page_size` value overrides the default page size.
+Controller administrators can also configure a `max_page_size` option that constrains the maximum page size a user can request.
```

Comment:
The workers list endpoint doesn't support the new pagination workflow, so we can remove these two

Reply:
Good catch, thank you!